robot_motion_planning_summary.tex

```diff
@@ -225,18 +255,122 @@ \subsection{Path and Map Smoothing}
 
 \section{Probabilistic Robotics}
 
-\td{noch nich fertig... \\ not yet written...}
+\td{noch nicht fertig... \\ not yet written...}
 \subsection{Kalman Filter}
 
+\paragraph{predict/ time update}
+
+\begin{align}
+	&\text{mean} && \hat{x}_{k+1}^- = A \hat{x}_{k} + B u_k  \\
```

Comment:
"state" instead of "mean"? (also in other places)

Reply:
In a Kalman filter you interpret your state as the mean of a normal distribution, so technically both would be right. Not sure what is more explanatory.

Reply:
The text you added makes it clearer to the reader, thanks.